INSTALL.md

```diff
@@ -101,9 +101,11 @@ To skip the secrets tasks, just pass:
     --skip-tags=secrets
 
 
+It is suggested that you do not skip the secret tasks because you need the master_key variable to create the Tokens which will be important later in this process. 
+
```

Comment:
I think that we should add a line saying that if you run ansible-playbook with the --skip-tags=secrets then you can not run it with a secrets.yml file from then on

Reply:
I think the playbook should be changed to not run the "secrets" task if there is no `master_key` defined.  I believe it's currently optional, but the backend can't really be set up without one so it should be required.